src/sentry/south_migrations/0388_auto__add_index_file_checksum.py

```diff
@@ -0,0 +1,1055 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding index on 'File', fields ['checksum']
+        db.create_index('sentry_file', ['checksum'])
```

Comment:
We can't run this in production.

This needs to be done with `CREATE INDEX CONCURRENTLY`. grep the repo for examples of other migrations adding indexes.